mods/cnc/maps/nod07c/nod07c.lua

```diff
@@ -0,0 +1,199 @@
+NodUnitsVehicles1 = { 'bggy', 'bggy', 'bike', 'bike' }
+NodUnitsVehicles2 = { 'ltnk', 'ltnk' }
+NodUnitsEngineers = { 'e6', 'e6', 'e6', 'e6' }
+NodUnitsRockets = { 'e3', 'e3', 'e3', 'e3' }
+NodUnitsGunners = { 'e1', 'e1', 'e1', 'e1' }
+NodUnitsFlamers = { 'e4', 'e4', 'e4', 'e4' }
+
+GDI1 = { units = { ['e1'] = 2 }, waypoints = { waypoint0.Location, waypoint1.Location, waypoint2.Location, waypoint8.Location, waypoint2.Location, waypoint9.Location, waypoint2.Location } }
+GDI2 = { units = { ['e1'] = 10, ['e2'] = 8, ['mtnk'] = 1, ['jeep'] = 1 }, waypoints = { waypoint12.Location, waypoint15.Location, waypoint0.Location } }
+GDI3 = { units = { ['jeep'] = 1 }, waypoints = { waypoint0.Location, waypoint1.Location, waypoint3.Location, waypoint4.Location, waypoint3.Location, waypoint2.Location, waypoint5.Location, waypoint6.Location, waypoint2.Location, waypoint7.Location } }
+MTANK = { units = { ['mtnk'] = 1 }, waypoints = { waypoint14.Location, waypoint5.Location } }
+
+GunboatPatrolPath = { GunboatLeft.Location, GunboatRight.Location }
+
+targetsKilled = 0
+
+AutoGuard = function(guards)
+	Utils.Do(guards, function(guard)
+		Trigger.OnDamaged(guard, function(guard, attacker)
+			if not guard.IsDead and not attacker.IsDead then
+				guard.AttackMove(attacker.Location)
```

Comment:
.Hunt() is in this situation a bad idea, as it will make the enemy go for the rest of your squad, after the vehicles are destroyed, rendering the distraction useless.